haddock-api/src/Haddock/Backends/LaTeX.hs

```diff
@@ -832,7 +832,7 @@ ppDataHeader _ _ = error "ppDataHeader: illegal argument"
 
 -- | Print an application of a DocName and two lists of HsTypes (kinds, types)
 ppAppNameTypes :: DocName -> [HsType DocNameI] -> [HsType DocNameI] -> Bool -> LaTeX
-ppAppNameTypes n ks ts unicode = ppTypeApp n ks ts ppDocName (ppParendType unicode)
+ppAppNameTypes n _ks ts unicode = ppTypeApp n [] ts ppDocName (ppParendType unicode)
```

Comment:
As part of making Haddock not render invisible kind arguments, I found myself needing to suppress pretty-printing of kinds here (as well as in `ppAppNamesTypes` in `Haddock.Backends.Xhtml.Decl`). Do you think it's worth refactoring `ppAppNameTypes` to remove these kind arguments altogether, or can you think of a reason we'd want to keep them around here?